src/main/java/org/jitsi/jicofo/Participant.java

```diff
@@ -543,7 +547,7 @@ public MediaSourceGroupMap getSourceGroups()
         MediaSourceGroupMap addedGroups = (MediaSourceGroupMap) added[1];
 
         // Mark as source owner
-        String roomJid = roomMember.getContactAddress();
+        Jid roomJid = roomMember.getJabberID();
```

Comment:
I think getContactAddress returns the JID in the MUC, while getJabberID return the user's own JID.